recipes/pip-audit/meta.yaml

```
@@ -0,0 +1,77 @@
+{% set name = "pip-audit" %}
+{% set version = "1.1.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pip-audit-{{ version }}.tar.gz
+  sha256: 61d772968b6ef644f43ecceace89665d28d7ea521a9390e59188c4189d580856
+
+build:
+  number: 0
+  skip: True  # [py<36]
```

```suggestion
  noarch: python
```
It seems like the recipe could be noarch otherwise.

---

Done.